### ISSUE TITLE
Bug Report: i have a problem

### ISSUE THREAD

**Issue by HAHahahahaKkNkn** [KadeDev/Kade-Engine#857]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.
Windows ('x64')
## What version of the game are you using? Look in the bottom left corner of the main menu.
KE 1.5.2 FNF 0.2.7.1
## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
Deprecations
## Please describe your issue. Provide extensive detail and images if possible.
Basically,lime test works but lime build will not work due to these deprecations
is there a workaround for this?
![image](https://user-images.githubusercontent.com/85860454/121840776-22f6d900-cd07-11eb-953d-1eba584d6767.png)

**Comment by prokube** [comment]
nothing bad there. just deprecations

are you sure its not building

**Comment by JustALilGuy** [comment]
> nothing bad there. just deprecations
> 
> are you sure its not building

I'm kinda having the same problem as him
Apparently lime **test** works but lime **build** doesn't due to these deprecations,or for other reasons that I do not know
